docs/css/style.css

```
@@ -188,6 +188,14 @@ img.screenshot {
   max-width: 100%;
 }
 
+.sponsor-logo {
```

It looks like this class name is on several adblocker lists. Ghostery put `display: none !important` on it on my machine, which I think is something we'd like to avoid.

```suggestion
#matomoLogo {
```